extensions/smallrye-openapi/deployment/src/main/java/io/quarkus/smallrye/openapi/deployment/SmallRyeOpenApiProcessor.java

```diff
@@ -271,6 +272,20 @@ LogCleanupFilterBuildItem logCleanup() {
                 "OpenAPI document initialized:");
     }
 
+    private boolean shouldScanAnnotations(Capabilities capabilities) {
+        // Disabled via config
```

Comment:
Fairly minor, but would it be better to save the result of this method so that subsequent calls don't need to re-evaluate everything?

Reply:
Ok cool, in a buildItem you mean ? Or a class var ?

Reply:
Just a class var unless it's needed outside the extension

Reply:
I did not think this is expensive though ? And it's only been called twice ?

Reply:
That's why I said it was minor.

Not a huge deal for now, but something to cleanup later

Reply:
Ok cool.

Reply:
If I do need to make a change for the exclusion I'll add this

Reply:
I'll add it to the TODO for the next version.